ql/src/java/org/apache/hadoop/hive/ql/io/parquet/vector/VectorizedListColumnReader.java

```diff
@@ -129,21 +131,16 @@ private boolean fetchNextValue(PrimitiveObjectInspector.PrimitiveCategory catego
   private void addElement(ListColumnVector lcv, List<Object> elements, PrimitiveObjectInspector.PrimitiveCategory category, int index) throws IOException {
     lcv.offsets[index] = elements.size();
 
-    // Return directly if last value is null
```

Comment:
i think make it simple ..addElement should add element .. fetchNextValue can be done by the caller in the loop.